src/pages/[platform]/build-a-backend/auth/multi-step-sign-in/index.mdx

```diff
@@ -0,0 +1,1820 @@
+                    // Switch on the next step to take appropriate actions.
+                    // If `signInResult.isSignedIn` is true, the next step
+                    // is 'done', and the user is now signed in.
+                }
+            },
+             error -> Log.e("AuthQuickstart", "Confirm sign in failed: " + error)
+      );
+} catch (Exception error) {
+    Log.e("AuthQuickstart", "Unexpected error: " + error);
+}
+```
+
+</Block>
+
+<Block name="Kotlin - Callbacks">
+
+```kotlin
+try {
+Amplify.Auth.confirmSignIn(
+          "challenge answer",
+          { result ->
+              if (result.isSignedIn) {
+                Log.i("AuthQuickstart","Confirm signIn succeeded")
+                } else {
+                    Log.i("AuthQuickstart", "Confirm sign in not complete. There might be additional steps: ${result.nextStep}")
+                    // Switch on the next step to take appropriate actions.
+                    // If `signInResult.isSignedIn` is true, the next step
+                    // is 'done', and the user is now signed in.
+                }
+          }
+      ) { error ->
+          Log.e("AuthQuickstart", "Confirm sign in failed: $error")
+      }
+} catch (error: Exception) {
+    Log.e("AuthQuickstart", "Unexpected error: $error")
+}
+```
+
+</Block>
+
+<Block name="Kotlin - Coroutines">
+
+```kotlin
+try {
+    val result = Amplify.Auth.confirmSignIn(
+        "challenge answer"
+    )
+    if (result.isSignedIn) {
+        Log.i("AuthQuickstart", "Confirm signIn succeeded")
+    } else {
+        Log.i("AuthQuickstart", "Confirm sign in not complete. There might be additional steps: ${result.nextStep}")
+        // Switch on the next step to take appropriate actions.
+        // If `signInResult.isSignedIn` is true, the next step
+        // is 'done', and the user is now signed in.
+    }
+} catch (error: Exception) {
+    Log.e("AuthQuickstart", "Unexpected error: $error")
+}
+```
+
+</Block>
+
+<Block name="RxJava">
+
+```java
+
+RxAmplify.Auth.confirmSignIn(
+                "challenge answer").subscribe(
+                result -> {
+                    if (result.isSignedIn()) {
+                        Log.i("AuthQuickstart", "Confirm signIn succeeded");
+                    } else {
+                        Log.i("AuthQuickstart", "Confirm sign in not complete. There might be additional steps: " + result.getNextStep());
+                        // Switch on the next step to take appropriate actions.
+                        // If `signInResult.isSignedIn` is true, the next step
+                        // is 'done', and the user is now signed in.
+                    }
+                },
+                error -> Log.e("AuthQuickstart", "Confirm sign in failed: " + error)
+        );
+```
+
+</Block>
+</BlockSwitcher>
+
+<Callout warning>
+
+<b>Special Handling on ConfirmSignIn</b>
```

Comment:
can these bolded callout notes be replaced with markdown?

Reply:
All done.